web-rdfs/source/http_handlers.cc

```diff
@@ -1,8 +1,15 @@
 // Copyright 2017 Rice University, COMP 413 2017
 
 #include "http_handlers.h"
+#include <cstdlib>
 #include <easylogging++.h>
 
+zkclient::ZkNnClient *zk;
```

Comment:
It feels strange to maintain a global here with a "setter" - is there a reason we can't leave `zk` in the server class as it is now and just pass a reference to each of the handlers as an argument?

Reply:
I don't think the api supports passing in any args besides response and request